components/mailbox/src/index.html

```diff
@@ -19,12 +19,17 @@
       }
     </style>
     <script type="module">
-      function toggleComposer(event, component) {
+      function toggleComposer(event, component, eventType) {
         component.removeAttribute("hidden");
         component.value = event.detail.value;
         if (Object.keys(event.detail.message).length) {
           component.message_with_body = event.detail.message;
         }
+        if (eventType === "replyClicked" || eventType === "replyAllClicked") {
+          component.focus_body_onload = true;
+        } else {
+          component.focus_body_onload = false;
+        }
```

Comment:
I think this should be included in the dispatched event rather than set here. WDYT?

Reply:
The event is dispatched from Email, while the values are being set here for Composer. We could consolidate all the necessary values to be set in an object within the event dispatched and set composer to that object. It would be a breaking change though, since Renegade is using this right now.

Reply:
Actually, I could just send another field `focus_body_onload` in the dispatched event and set it here. I guess that's what you meant. Thank you for suggesting that.

Reply:
Updated 👍🏽